polyfills/fetch/config.json

```diff
@@ -0,0 +1,24 @@
+{
+	"aliases": [
+		"es6"
```

Comment:
`fetch` is not a part of ES6. It's a [host object](https://fetch.spec.whatwg.org/) like WebSoket, FormData etc.